python/cac_tripplanner/destinations/views.py

```diff
@@ -47,10 +47,10 @@ def home(request):
     }
     if request.GET.get('destination') is not None:
         # If there's a destination in the URL, go right to directions
-        context['tab'] = 'map'
+        context['body_class'] = 'body-map body-map-dir body-has-sidebar-banner'
```

Comment:
The tricky thing is that if the page class in the template is 'home', the Home view mostly renders by the time the javascript runs and switches it to the desired view, which looks pretty bad.  And the rendering of "no body class" isn't really defined, so that doesn't look good either.

If every cold load was supposed to start on Home and then animate into the target view, that would be another story.  But even if we add nice animations, I think loading a direct link by animating to it would look weird.

I can poke at it some more and see if I can figure out a better approach.  Thoughts and suggestions welcome :smile: